src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -265,6 +265,61 @@ public void testBedIndex() {
         Assert.assertEquals(index.getSequenceNames(), Arrays.asList("1", "2", "4"));
     }
 
+    // TODO: this is not enabled because canDecode returns false for bed.gz extension
+    // TODO: should this be changed at the htsjdk level to return true also for a block-compressed extension?
```

Comment:
Yes, you're right -- assuming that `AbstractFeatureReader.getFeatureReader()` will work correctly in the block-gzipped case with this codec, `canDecode()` should be returning true. Recommend filing a ticket or PR against htsjdk to fix this.